src/test/java/api/support/APITestContext.java

```diff
@@ -55,6 +55,10 @@ public static String getUserId() {
     return USER_ID;
   }
 
+  public static void setUserId(String userId) {
```

Comment:
I'm not keen on altering the global test state like this, however I understand there isn't really another mechanism at this point.